wire-grpc-tests/src/test/proto-grpc/routeguide/RouteGuide.kt

```diff
@@ -1,28 +1,15 @@
-/*
- * Copyright 2019 Square Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: routeguide/RouteGuideProto.proto
```

Comment:
accidental deletion?

Reply:
Nop, it's now being generated by this PR